web/v1/bigquery/sql-runner/sql/standard/01-base/01-main/03-sessions-to-process.sql

```diff
@@ -40,5 +40,13 @@ AS(
     AND a.domain_sessionid IS NOT NULL
     AND TIMESTAMP_DIFF(a.dvce_sent_tstamp, a.dvce_created_tstamp, DAY) <= {{or .days_late_allowed 3}}
     -- don't process data that's too late
+
+    {{if eq (or .derived_tstamp_partitioned false) true}}
```

Comment:
Is it a bit odd that these become limited by both collector and derived when you toggle this on? Should it be one or the other?

Reply:
Since the manifests and all logic later on depend on collector_tstamp, it'd be a lot more changes to just use derived, since all the manifests etc depend on collector. ( I'm not even sure derived can be reliably used alone, since it depends on device clock).

Am adding this change because it's a simple way to allow backwards compatibility for older customers who still partition by derived tstamp (default is collector tstamp now, and officially that's what the model supports). If it involves further change then it's not worth including at all IMO.

Reply:
That seems fair to me!